client/my-sites/themes/theme-showcase.jsx

```diff
@@ -24,10 +24,6 @@ import { getSiteSlug } from 'state/sites/selectors';
 import { getCurrentUserId } from 'state/current-user/selectors';
 import ThemePreview from './theme-preview';
 import config from 'config';
-import getThemeFilterTerms from 'state/selectors/get-theme-filter-terms';
-import getThemeFilterToTermTable from 'state/selectors/get-theme-filter-to-term-table';
-import getThemeShowcaseDescription from 'state/selectors/get-theme-showcase-description';
-import getThemeShowcaseTitle from 'state/selectors/get-theme-showcase-title';
 import prependThemeFilterKeys from 'state/selectors/prepend-theme-filter-keys';
```

Comment:
This seems like another selector that we could move the same way that we did with others.

Reply:
Done!